topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -732,14 +732,15 @@ Here, treatment is the primary factor that we are interested in. The sequencing
 >
 > 3. Rename the datasets (if needed)
 > 4. **DESeq2** {% icon tool %} with the following parameters:
->    - *"how"*: `Select group tags corresponding to levels`
+>    - *"how"*: `Select datasets per level`
 >       - In *"Factor"*:
 >          - In "1: Factor"
 >              - *"Specify a factor name"*: `Treatment`
 >              - In *"Factor level"*:
 >                  - In *"1: Factor level"*:
 >                      - *"Specify a factor level"*: `treated`
 >                      - {% icon param-files %} *"Counts file(s)"*: the 3 gene count files with `treat` in their name
+>                         - Tip: use the CTRL key to select multiple files
```

Comment:
```suggestion
>                         - Tip: use the <kbd>Ctrl</kbd> key to select multiple files
```

Reply:
thanks :)

Reply:
you could also add the tip box here for multiple dataset selection

Reply:
oh, good idea, didn't know we had a snippet for that :)